nss-day-cohort-47/tabloidfullstack-bingo
# Hp view catagories

# Description
Displays list of categories when user selects the categories tab.

## Type of change
Please delete options that are not relevant.
# Testing Instructions
Log in and navigate to categories page, a list of categories should be displayed.
# Checklist:
